common/men_at_arms_types/wc_cultural_maa_types.txt

```diff
@@ -0,0 +1,1689 @@
+﻿# standard costs
+@maa_buy_cost = 150
+@maa_low_maintenance_cost = 1.0
+@maa_high_maintenance_cost = 5.0
+@cultural_maa_extra_ai_score = 60 # Equivalent to having 6 extra regiments beyond what the code scoring would indicate (see NEGATIVE_SCORE_PER_EXISTING_REGIMENT)
+
+abomination = {
+	type = massive_infantry
+	
+	damage = 60
+	toughness = 80
+	pursuit = 45
+	screen = 35
+	
+	terrain_bonus = {
+		wetlands = { damage = -8 }
+		jungle = { damage = -10 }
+	}
+
+	counters = {
+		skirmishers = 1
+		light_aerial = 1
+	}
+	
+	buy_cost = { gold = 120 }
+	low_maintenance_cost = { gold = massive_infantry_low_maint_cost }
+	high_maintenance_cost = { gold = massive_infantry_high_maint_cost }
+	
+	stack = 50
+	ai_quality = { value = @cultural_maa_extra_ai_score }
+	icon = abomination
+}
+
+archmage = {
+	type = caster
+	
+	damage = 70
+	toughness = 15
+	pursuit = 15
+	screen = 15
+	
+	terrain_bonus = {
+		hills = { toughness = 5 }
+		plains = { damage = 5 pursuit = 5 }
+		mountains = { damage = 5 }
+		drylands = { damage = 5 pursuit = 5}
+	}
+
+	counters = {
+		pikemen = 1
+		massive_infantry = 1
+		heavy_infantry = 1
+	}
+
+	buy_cost = { gold = 120 }
+	low_maintenance_cost = { gold = caster_low_maint_cost }
+	high_maintenance_cost = { gold = caster_high_maint_cost }
```

Comment:
You need to create real values, somewhere in `common\script_values` just like it's in the vanilla.

Reply:
instead of what? the buy cost value?  Wouldn't it just do the same thing but add a bunch of extra lines?

Reply:
> instead of what? the buy cost value? Wouldn't it just do the same thing but add a bunch of extra lines?

The maintenance costs, buy cost. The maintenance costs are calculated in-game.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/127228695-1f003b12-172c-4b24-9259-7fc38787017f.png)

</details>

Reply:
![image](https://user-images.githubusercontent.com/74790101/127418270-2ef03072-0d6e-4379-8484-cb4a28ebf5b1.png)

I mean do you really want me to do this for every single unit?

Reply:
> I mean do you really want me to do this for every single unit?

Yes, except it will look like this.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/127550116-aa1b8a8d-211b-4f98-ae74-5e4fcb9b1876.png)

</details>

Reply:
@zumbak04 This doesnt seem to work

![image](https://user-images.githubusercontent.com/74790101/127751343-1eff6345-afc9-45f9-8916-c0cfc94cd174.png)
![image](https://user-images.githubusercontent.com/74790101/127751346-0c0c4fe7-6521-447e-bdbc-3fb435c4831e.png)